lib/typing/ext/variables.go

```diff
@@ -37,7 +38,8 @@ var SupportedTimeFormats = []string{
 
 // RFC3339 variants
 const (
-	RFC3339Millisecond = "2006-01-02T15:04:05.000Z07:00"
-	RFC3339Microsecond = "2006-01-02T15:04:05.000000Z07:00"
-	RFC3339Nanosecond  = "2006-01-02T15:04:05.000000000Z07:00"
+	RFC3339MillisecondUTC = "2006-01-02T15:04:05.000Z"
+	RFC3339Millisecond    = "2006-01-02T15:04:05.000Z07:00"
+	RFC3339Microsecond    = "2006-01-02T15:04:05.000000Z07:00"
```

Comment:
Should we also add `RFC3339MicrosecondUTC` and `RFC3339Nanosecond UTC`?

Reply:
Doesn't hurt, I'll add!